src/test-run/execute-js-expression.js

```diff
@@ -0,0 +1,19 @@
+import { createContext, runInContext } from 'vm';
+import SelectorBuilder from '../client-functions/selectors/selector-builder';
+
+export function executeSelectorExpression (expression, skipVisibilityCheck) {
+    var sandbox = {
+        Selector: (fn, options = {}) => {
+            if (skipVisibilityCheck)
+                options.visibilityCheck = false;
+
+            var builder = new SelectorBuilder(fn, options, { instantiation: 'Selector' });
```

Comment:
Nope, we call this `execute` function from the same `try-catch` block: https://github.com/DevExpress/testcafe/pull/1573/files#diff-5998664e6df837cb69bedb1efa14a399R37